### PR TITLE
Bug 1403344 - Add command to docker container for schema / data migra…

### PR DESCRIPTION
**Pull request by dylanwh**
…tions

### CODE REVIEW

**Comment by dylanwh** [comment]
I had to actually generate a fake schema change to test this, very short change. 
It does work. :-)

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,5 @@
 FROM mozillabteam/bmo-slim:20170927.1
 
-ENV BUNDLE=https://s3.amazonaws.com/moz-devservices-bmocartons/bmo/vendor.tar.gz
```
Comment (floatingatoll) [inline Dockerfile]: This change isn't documented as necessary anywhere in the PR or in the bug, but it's mixed in with a documented change below to add checksetup. Is this an intentional change?

Comment (dylanwh) [reply Dockerfile]: I'll add it back, it isn't used by anything but it was left over from another session.

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 FROM mozillabteam/bmo-slim:20170927.1
 
+
```
Comment (globau) [inline Dockerfile]: don't forget to remove this

**Pull request merged**